nodejs/repositories.bzl

```diff
@@ -472,6 +472,7 @@ def nodejs_register_toolchains(name = DEFAULT_NODE_REPOSITORY, register = True,
         name = name + "_host",
         user_node_repository_name = name,
     )
+
```

Comment:
How did this get into this PR?

Reply:
Related cleanup to the test in the repository rule the tests are testing